docs/source/tutorials/index.rst

```diff
@@ -0,0 +1,7 @@
+Tutorials
+============
+
+.. toctree::
+    :maxdepth: 2
+
+    install
```

Comment:
It would probably make sense to have a `Getting Started` Section, of which Installation is a part?

Reply:
What else would be a part of that section?

Reply:
In the future

Reply:
Hmm no, I'm just unsure if tutorials is the right place for install. We could have more useful things there like
1. How to create a new bandit
2. How to create a new Deep agent
etc

Reply:
Getting Started could just have, Install and few small examples (e.g. the ones in readme currently)?

Reply:
Since Getting started section will be pretty small, we could have Installation, Getting Started and say, Contributing as separate pages in the index, along with the Tutorials (with what you said) and Algorithms.

Reply:
Great, makes sense to me.